test/specs/testnet/swap/Smoke_CrossChainQuote.spec.js

```diff
@@ -0,0 +1,4195 @@
+import * as dotenv from "dotenv";
```

Comment:
Assuming this file is nearly the same as the file above but in the mainnet folder, so copying and pasting the same information from that file below:

This file is 4000+ lines long. This absolutely must be split into smaller files as i can see that a lot of the code is being repeated and doesn't need to all be in one file. In summary, the following were observed:

- Please use the correct messages in the assertion messages. For example, if the assertion is isNumber please reflect that the assertion failed message was because the value was not a number
-  Please use correct variable names for sdkMainNet - this refers to the Ethereum network. Use something more specific like sdkArbitrum
- Remember that all SDK methods may throw an error especiallt the estimation and send methods. These need to be asserted also
- Remove any dead code unless it's there for a reason, then please comment why it is there
- Remember that we will run these tests many many times, so please lower the amounts we are using to swap unless that amount is required by the test for a reason - in which case it should be commented